package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I know it's not required to install the dependencies, but can you add:
* `name
* `description`
* `license`
* `repository`
to the package.json file?